LICENSE.rst

```diff
@@ -1,3 +1,9 @@
+BSD 3-Clause License
```

Comment:
This is the wrong way around.  This is not a BSD 3 clause license so we must remove this.

Simply use the licesnse from the old code, but  add a 2024 columbia copyright.